code/src/components/Like.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+// needs individual settings
+
+/* export const Like = () => {
+  fetch('https://happy-thoughts-ux7hkzgmwa-uc.a.run.app/thoughts', {
+    method: 'POST',
+    body: JSON.stringify()
```

Comment:
Looks like you are already aware of this needing to be removed since all the code is commented out